cp-styles/app/styles/components/_grid-field.scss

```diff
@@ -17,7 +17,40 @@
   }
 
   tr {
-
+  	td {
+  		min-width: 250px;
+  		max-width: 400px;
+
+  		@include m-tablet-down {
+  			min-width: 0 !important;
+  			width: 100% !important;
+  			max-width: 100% !important;
+  		}
+
+  		&.grid-rte,
+  		&.grid-file-upload,
+  		&.grid-multi-relate,
+  		&[data-fieldtype="wygwam"] {
```

Comment:
While we own `wygwam`, technically it's third-party add-on, so if possible we should not reference it in EE stylesheets and use something more generic instead. If it's something specific for Wygwam, it should be added to Wygwam own styles instead.
Also, does this code mean we're limiting max width to 600 px even if the Grid has only one column?

Reply:
no, as you can see on the screenshot it takes full space
![Edit Entry Sub Page Two ExpressionEngine 2022-01-10 at 1 20 52 PM](https://user-images.githubusercontent.com/23382425/148758420-3921cd2a-d739-4184-b8db-a13c3a1bd5f3.jpg)

![Edit Entry Sub Page Two ExpressionEngine 2022-01-10 at 1 23 16 PM](https://user-images.githubusercontent.com/23382425/148758394-9739c063-c59a-4104-9925-a365c4a41b30.jpg)